CSSスタイル解析修正: インライン style属性パース不整合 (#1085)

## 問題
CI失敗: `test_attribute_parser_comprehensive.py` でCSSスタイル解析エラー

## エラー詳細
```
FAILED test_inline_style_property_parsing
AssertionError: assert 'color: red;' == 'color:'
```

## 原因
- インラインスタイルのプロパティ値が正しく解析されていない
- パーサーが値部分を無視している可能性

## 修正方針
1. CSSプロパティパーサーの修正
2. 値部分の取得ロジック改善

## 優先度
高（テスト失敗）

--- a/tests/unit/parsing/keyword/test_attribute_parser_comprehensive.py
+++ b/tests/unit/parsing/keyword/test_attribute_parser_comprehensive.py
@@ -223,17 +223,17 @@ class TestAttributeParserCSS:
         # 基本スタイル
         content = 'style="color: red;"'
         attributes = self.parser.parse_attributes_from_content(content)
-        assert attributes["style"] == "color:"  # パターンマッチング結果
+        assert attributes["style"] == "color: red;"  # 完全なスタイル値
 
         # 複数プロパティ
         content = 'style="color: red; background: blue; margin: 10px;"'
         attributes = self.parser.parse_attributes_from_content(content)
-        assert "style" in attributes
+        assert attributes["style"] == "color: red; background: blue; margin: 10px;"
 
         # 複雑な値
         content = 'style="background: linear-gradient(45deg, red, blue);"'
         attributes = self.parser.parse_attributes_from_content(content)
-        assert "style" in attributes
+        assert attributes["style"] == "background: linear-gradient(45deg, red, blue);"
 
     def test_css_selector_compatibility(self):
         """CSSセレクター互換性テスト"""

